lib/pages/courses/components/course_card.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:youniversity_app/pages/courses/models/course_model.dart';
+import 'package:youniversity_app/utils/build_context_extensions.dart';
+import 'package:youniversity_app/utils/widget_list_extensions.dart';
+import 'package:youniversity_app/utils/text_style_extensions.dart';
+
+class CourseCard extends StatelessWidget {
+  final CourseModel course;
+
+  const CourseCard({
+    super.key,
+    required this.course,
+  });
+
+  Widget courseInformation(BuildContext context, String label, IconData icon) {
```

Comment:
Could you name this `createCourseInformation`?

```suggestion
  Widget createCourseInformation(BuildContext context, String label, IconData icon) {
```